### DISCUSSION TITLE
Pagination for Workflows missing `continue` unlike CronWorkflows?

### DISCUSSION THREAD

**Discussion by geraldbauer** [argoproj/argo-workflows#13289] [Q&A] [answered]
I understand how Argo cron-workflow pagination works however I am perplexed on workflow pagination. 

For cron-workflow my URI is `//{{uri}}/api/v1//cron-workflows/{{namespace}}?listOptions.limit=2` which yields expected results. Within the results' `metadata` we have a property called `continue` that has a value that can be used for the next page - `//{{uri}}/api/v1//cron-workflows/{{namespace}}?listOptions.limit=2&listOptions.continue=eyJ2IjoibWV0YS5rOHMuaW8vdjEiLCJydiI6MTk3NDkyOTUsInN0YXJ0IjoiZ2VyYWxkLWxlYXJuaW5nLWFnZW50MjAyMy0xMHBtXHUwMDAwIn0`

For workflow my URI is `//{{uri}}/api/v1//workflows/?listOptions.limit=2` which yields expected results. I am not able to locate within the results' `metadata` the property `continue` for my next page. 

Where should I look for the `continue` property within the results?

**Comment by agilgur5** [comment, marked as answer]
It should be the same (they use the same interfaces, in this case the k8s `metav1.ListMeta`), but `continue` is not present if there are no more remaining items.

Also what version of Argo are you on? 3.5 prior to 3.5.7 did have a pagination bug: #11715

**Comment by geraldbauer** [comment]
Thank you. We are using 3.5. I will update to 3.5.7.

**Reply by agilgur5** [reply]
Should do 3.5.8 as it is latest patch (and includes a critical fix to a 3.5.7 bug: #13166)
